node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -96,6 +96,7 @@ impl NodeConfigurator<BootstrapperConfig> for NodeConfiguratorStandardUnprivileg
         configure_database(&unprivileged_config, persistent_config.as_mut())?;
         let cryptde_pair = if multi_config.occurrences_of("fake-public-key") == 0 {
             configure_cryptdes(
+                None,
```

Comment:
This is a new parameter. `Some(true)` for `new-public-key on`, `Some(false)` for `new-public-key off`, and `None` for no mention of `new-public-key`.